src/uwtools/config/support.py

```diff
@@ -88,3 +89,17 @@ def represent(dumper: yaml.Dumper, data: TaggedString) -> yaml.nodes.ScalarNode:
         documentation for details.
         """
         return dumper.represent_scalar(data.tag, data.value)
+
+
+def represent_namelist(dumper: yaml.Dumper, data) -> yaml.nodes.MappingNode:
+    """
+    Serialize f90nml tags.
+    """
+    # Convert the f90nml Namelist object to a dictionary
```

Comment:
The command says `Namelist` object (which is a real type from the `f90nml` library), but the `isinstance()` call checks for `OrderedDict`.

Reply:
Oof thanks, I was rushing through that